plugins/services/src/js/containers/tasks/TasksView.js

```diff
@@ -225,19 +188,46 @@ class TasksView extends mixin(SaveStateMixin) {
     );
   }
 
-  render() {
-    const { inverseStyle, tasks } = this.props;
-    const { checkedItems, filterByStatus, searchString } = this.state;
-    let filteredTasks = this.getFilteredTasks();
+  getFilterBar() {
+    const {
+      filters,
+      filterExpression,
+      handleExpressionChange,
+      defaultFilterData
+    } = this.props;
+
+    const hostClasses = classNames({
+      "column-medium-5": !filterExpression.value,
```

Comment:
`const filterExpression = filterExpression.value`; then you can re-use. Not big deal though.